suite-common/wallet-core/src/transactions/__fixtures__/transactionsReducer.ts

```diff
@@ -149,3 +149,30 @@ export const addTransaction = [
         result: { [ACCOUNT.key]: [testMocks.getWalletTransaction({ txid: '00' })] },
     },
 ];
+
+export const removeTransaction = [
```

Comment:
Please lets type all fixtures decoratively. Without it, its hard to debug failing test wen we add something to action payload

Reply:
ok fascist